snippets/security-catalog.mdx

```diff
@@ -4,4 +4,6 @@ protect your applications from a wide range of threats, ensuring your users' dat
 <Columns cols={2}>
   {/* Bot Protection */}
   <Card title="DataDome" href="/components/security/datadome" horizontal={true} icon={<img src="https://mintlify.s3.us-west-1.amazonaws.com/edgee/images/logos/datadome.svg" alt="DataDome" />} />
+  {/* IP Blocking */}
+  <Card title="CrowdSec" href="/components/security/crowdsec" horizontal={true} icon={<img src="https://mintlify.s3.us-west-1.amazonaws.com/edgee/images/logos/crowdsec.svg" alt="CrowdSec" />} />
```

Comment:
not sure why this file wasn't updated yet, but this can just be:
```suggestion
  <Card title="CrowdSec" href="/components/security/crowdsec" horizontal={true} icon="/images/logos/crowdsec.svg" />
```

can you please do the same for DataDome above?